sechub-web-server/src/main/java/com/mercedesbenz/sechub/webserver/encryption/AES256Encryption.java

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.webserver.encryption;
+
+import java.nio.charset.StandardCharsets;
+import java.security.GeneralSecurityException;
+import java.util.Base64;
+
+import javax.crypto.Cipher;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@EnableConfigurationProperties(AES256EncryptionProperties.class)
+public class AES256Encryption {
+
+    private static final Logger LOG = LoggerFactory.getLogger(AES256Encryption.class);
+    private static final String TRANSFORMATION = "AES";
+    private static final Base64.Encoder ENCODER = Base64.getEncoder();
+    private static final Base64.Decoder DECODER = Base64.getDecoder();
+
+    private final Cipher encrypt;
+    private final Cipher decrypt;
+
+    AES256Encryption(AES256EncryptionProperties properties) throws GeneralSecurityException {
+        SecretKey secretKey = new SecretKeySpec(properties.getSecretKeyBytes(), TRANSFORMATION);
+        this.encrypt = Cipher.getInstance(TRANSFORMATION);
+        encrypt.init(Cipher.ENCRYPT_MODE, secretKey);
+        this.decrypt = Cipher.getInstance(TRANSFORMATION);
+        decrypt.init(Cipher.DECRYPT_MODE, secretKey);
+    }
+
+    public String encrypt(String plainText) {
```

Comment:
The encrypted text must be here always base64 encoded. If we do not separate the encoding and the encryption, the parameter should be renamed to `base64plainText` IMO.

Also the method should be rneamed to `encryptBase64`.

Remark: But I think separate encoding and encryption, would make things clearer, easier etc.